bom/runtime/src/main/resources/extensions-support-overrides.json

```diff
@@ -160,6 +184,14 @@
          "dev-support" ]
        }
     },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-logging-json",
```

Comment:
@tqvarnst mentioned that he might shift it to TP or unsupported level

Reply:
@rsvoboda let me know, create a new issue in that case ;-)

Reply:
quarkus-logging-json stays on supported level